aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -23,33 +35,30 @@
 }
 
 app.MapGet("/", () => "Hello World!");
+app.MapGet("/admin", () => "Authorized Endpoint")
+    .RequireAuthorization("AdminsOnly"); ;
 
-// todo endpoints
-var todos = app.MapGroup("/todos").WithTags("Todo Endpoints").AddRouteHandlerFilter(async (context, next) =>
-{
-    app.Logger.LogInformation("Accessing todo endpoints");
-    return await next(context);
-});
-todos.MapGet("/", RouteHandlers.GetAllTodos);
-todos.MapGet("/{id}", RouteHandlers.GetTodo);
-todos.MapPost("/", RouteHandlers.CreateTodo).AddRouteHandlerFilter(async (context, next) =>
-{
-    // log time taken to process
-    var start = DateTime.Now;
-    var result = await next(context);
-    var end = DateTime.Now;
-    app.Logger.LogInformation($"{context.HttpContext.Request.Path.Value} took {(end - start).TotalMilliseconds}ms");
-    return result;
-});
-todos.MapPut("/{id}", RouteHandlers.UpdateTodo).AddRouteHandlerFilter(async (context, next) =>
-{
-    // log time taken to process
-    var start = DateTime.Now;
-    var result = await next(context);
-    var end = DateTime.Now;
-    app.Logger.LogInformation($"{context.HttpContext.Request.Path.Value} took {(end - start).TotalMilliseconds}ms");
-    return result;
-});
-todos.MapDelete("/{id}", RouteHandlers.DeleteTodo);
+// todoV1 endpoints
+app.MapGroup("/todos/v1")
+   .MapTodosApiV1()
+   .WithTags("Todo Endpoints")
+   .AddEndpointFilter(async (context, next) =>
+   {
+       app.Logger.LogInformation("Accessing todo endpoints");
+       return await next(context);
+   });
```

Comment:
Are we planning showcase filters here as well?

Reply:
I think we can exclude filters here